docs/CSBR.md

```diff
@@ -857,22 +859,20 @@ Private Keys corresponding to CA Keys MUST be stored in accordance with BR Secti
 
 #### 6.2.7.2 Private key storage for Timestamp Authorities
 
-A Timestamp Authority MUST protect its signing key using a process that is at least to FIPS 140-2 Level 3, Common Criteria EAL 4+ (ALC\_FLR.2), or higher. The CA MUST protect its signing operations in accordance with the CA/Browser Forum's Network Security Guidelines.
+A Timestamp Authority MUST protect its signing key using a process that is at least to FIPS 140-2 Level 3, Common Criteria EAL 4+ (ALC\_FLR.2), or higher. The Timestamp Authority MUST protect its signing operations in accordance with the CA/Browser Forum's Network Security Guidelines.
 
 #### 6.2.7.3 Private key storage for Signing Services
 
-The Signing Service MUST ensure that a Subscriber's Private Key is generated, stored, and used in a secure environment that has controls to prevent theft or misuse. A Signing Service MUST enforce multi-factor authentication to access and authorize Code Signing and obtain a representation from the Subscriber that they will securely store the tokens required for multi-factor access. A system used to host a Signing Service MUST NOT be used for web browsing. The Signing Service MUST run a regularly updated antivirus solution to scan the service for possible virus infection. The Signing Service MUST comply with the Network Security Guidelines as a "Delegated Third Party".
+The Signing Service MUST ensure that a Subscriber's Private Key is generated, stored, and used in a secure environment that has controls to prevent theft or misuse. A Signing Service MUST enforce multi-factor authentication or secure server-to-server authentication to access and authorize the signing of Application Code. The Signing Service MUST comply with the Network Security Guidelines.
 
-For Code Signing Certificates, Signing Services shall protect Private Keys in a Hardware Crypto Module conforming to at least FIPS 140-2 level 2 or Common Criteria EAL 4+.
+For Code Signing Certificates, Signing Services SHALL protect Private Keys in a Hardware Crypto Module conforming to at least FIPS 140-2 level 3 or Common Criteria EAL 4+.
 
-Techniques that MAY be used to satisfy this requirement include:
+Techniques that MUST be used to satisfy this requirement include:
 
-1.  Use of an HSM, verified by means of a manufacturer's certificate;
+1.  Use of an HSM, verified by means of the FIPS or Common Criteria certificate; or

```

Comment:
Is this correct?
The way I read the old language, this is meant as key attestation. However the new language doesn't, and I'm not sure providing a FIPS certification document proofs that a HSM was used for key generation itself

Reply:
I did not understand key attestation, but understood certification, so a certificate would show certification.

Reply:
I agree with @XolphinMartijn a FIPs certification proves you have a FIPs HSM but doesn't ensure the key was generated there. Is the desire to change the language from Manufacturers certificate because there is a lack of standard methods to request that from the HSM? That has been my experience.

Reply:
Right this change effectively removes the previous requirement, as FIPS is always required.